jsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can you include a comment that this file is for Visual Studio Code? It is probably obvious to VSC users, but for non-MS folks, it might be helpful.